docs/proposals/formatted-output.md

```diff
@@ -321,3 +320,8 @@ oras discover localhost:5000/hello:v1 --format json
 
 **Q:** Why ORAS chooses [Go template](https://pkg.go.dev/text/template)?
 **A:** Go template is a powerful method to customize output you want It allows users to manipulate the output format of certain commands. It provides access to data objects and additional functions that are passed into the template engine programmatically. It also has some useful libraries that have strong functions for Go’s template language to manipulate the output data, such as [Sprig](https://masterminds.github.io/sprig/).
+
+**Q:** What is the difference of prettified JSON and raw JSON?
+**A:** In the context of ORAS output, raw JSON means display the output of ORAS command in the JSON string format, while prettified JSON means display the output of ORAS command in a pretty format. Only necessary fields will be displayed in the formatted JSON output. See examples in the [oras pull](#oras-pull) section.
```

Comment:
This answer does not answer me the question, and I'm still confused after reading it.

To me, _raw JSON_ means that `oras` outputs the JSON object as it is returned by the server. _Prettified JSON_ means that `oras` outputs the modified JSON object with whitespace adjusted (i.e. human-friendly indentations and newlines).

Reply:
Good suggestion!